src/GuiBase/BaseApplication.cpp

```diff
@@ -215,6 +216,12 @@ BaseApplication::BaseApplication( int argc,
     m_engine->initialize();
     addBasicShaders();
 
+    // Register the GeometrySystem converting loaded assets to meshes
+    m_engine->registerSystem( "GeometrySystem", new Ra::Engine::GeometrySystem, 1000 );
```

Comment:
What is the meaning of the constant 1000 ?

Reply:
That is system priority.